IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/SSNewsCollectionView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SSNewsCollectionView.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+final class SSNewsCollectionView: UICollectionView, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    var items: [SSNewsCollectionCellVM] = []
+    weak var viewModelDelegate: SSNewsCollectionViewModel?
+    var cellIdentifier: String?
```

Comment:
Yes, I had started to create the class more generic but little by little I was changing. I will remove the "cellIdentifier"